docs/common/craft-parts/explanation/parts.rst

```diff
@@ -165,12 +159,18 @@ system packages that need to be deployed with the part are specified using
 the :ref:`stage_snaps` and :ref:`stage_packages` properties. Files to be
 deployed are specified using the :ref:`stage` property.
 
+.. ifconfig:: project in ("Rockcraft",)
```

Comment:
This shouldn't be scoped to Rockcraft - Snapcraft also supports chisel slices I believe